draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1421,6 +1421,12 @@ SDP Answer
       </t>
     </section>
 
+    <section title="RTP Header Extensions Consideration" anchor="sec-extmap" toc="default">
+      <t>When <xref target="RFC5285"/> RTP header extensions are used in the context of this 
+       specification, the identifier used for a given extension MUST identify the the same 
+       extension across all the bundled media descriptions. </t>

```

Comment:
I'd say "all media descriptions in the same BUNDLE group", since technically there may be multiple BUNDLE groups, and they could use different identifier spaces.